plugins/BEdita/Core/src/Model/Entity/Stream.php

```diff
@@ -248,4 +254,16 @@ protected function _getUrl()
 
         return $this->_properties['url'] = FilesystemRegistry::getPublicUrl($this->uri);
     }
+
+    protected function _getMetadata($source)
+    {
+        $ext_allowed = Configure::read('Streams.ext_allowed');
+        if (in_array($this->mime_type, $ext_allowed)) {
+            try {
+                $this->file_metadata = json_encode(exif_read_data($source));
+            } catch (\Exception $e) {
+                return null;
```

Comment:
add `$this->log(...., 'warning');` to track this event as warning - use `uuid` and `filename` in message